doozer/doozerlib/distgit.py

```diff
@@ -1306,50 +1312,74 @@ def update_konflux_db(self, build_info, outcome, build_pipeline_url='', scratch=
             self.logger.warning('Konflux DB connection is not initialized, not writing build record to the Konflux DB.')
             return
 
-        self.logger.info('Storing build info in Konflux for Brew build %s', build_info['nvr'])
         try:
             dfp = DockerfileParser(str(self.dg_path.joinpath('Dockerfile')))
             source_repo = self.metadata.config.content.source.git.url
             commitish = dfp.labels['io.openshift.build.commit.url'].split('commit/')[-1]
 
-            rebase_repo_url, rebase_commitish = build_info['source'].split('#')
-
-            image_pullspec = build_info['extra']['image']['index']['pull'][0]
+            _, rebase_repo_url, _ = gather_git(['-C', self.distgit_dir, 'remote', 'get-url', 'origin'])
+            _, rebase_commitish, _ = gather_git(['-C', self.distgit_dir, 'rev-parse', 'HEAD'])
 
             try:
                 release_target = self.org_release.split('.')[-1]
             except KeyError:
                 release_target = ''
             el_target = release_target if release_target else f'el{self.metadata.branch_el_target()}'
 
-            build_record = KonfluxBuildRecord(
-                name=self.metadata.name,
-                group=self.runtime.group,
-                version=build_info['version'],
-                release=build_info['release'],
-                assembly=self.runtime.assembly,
-                el_target=el_target,
-                arches=self.metadata.get_arches(),
-                installed_packages=self.get_installed_packages(image_pullspec),
-                parent_images=[build['nvr'] for build in build_info['extra']['image']['parent_image_builds'].values()],
-                source_repo=convert_remote_git_to_https(source_repo),
-                commitish=commitish,
-                rebase_repo_url=rebase_repo_url,
-                rebase_commitish=rebase_commitish,
-                embargoed='p1' in build_info['release'].split('.'),
-                start_time=datetime.strptime(build_info['start_time'], '%Y-%m-%d %H:%M:%S.%f'),
-                end_time=datetime.strptime(build_info['completion_time'], '%Y-%m-%d %H:%M:%S.%f'),
-                artifact_type=ArtifactType.IMAGE,
-                engine=Engine.BREW,
-                image_pullspec=image_pullspec,
-                image_tag=image_pullspec.split('sha256:')[-1],
-                outcome=outcome,
-                art_job_url=os.getenv('BUILD_URL', 'n/a'),
-                build_pipeline_url=build_pipeline_url,
-                pipeline_commit='n/a',
-                nvr=build_info['nvr'],
-                build_id=str(build_info['id'])
-            )
+            if outcome == KonfluxBuildOutcome.FAILURE:
+                self.logger.info('Storing failed Brew build info for %s in Konflux DB', self.metadata.name)
+                build_record = KonfluxBuildRecord(
```

Comment:
Thinking it would be good to DRY this, so
```python
build_record_params = {name: self.metadata.name, ..  }  # all the common params
if outcome == success:
  build_record_params.update(..)
else:
  build_record_params.update(..)
build_record = KonfluxBuildRecord(**build_record_params)
```

Reply:
Done in fcd7faf8481e280412233b8051c9d22ff251b722